types.d.ts

```diff
@@ -1,24 +1,41 @@
 import { bits } from 'discord-rose/dist/utils/Permissions'
-import { CommandContext } from "discord-rose/dist/structures/CommandContext";
+import { CommandContext } from "discord-rose/dist/typings/lib";
```

Comment:
To stay consistent with formatting, the double quotes should be swapped with single quotes and the semicolon should be removed. This was not caused by the changes made in this pull request but since you are already changing things, you might as well just change the formatting.